datafusion/physical-plan/Cargo.toml

```diff
@@ -70,6 +70,7 @@ tokio = { workspace = true }
 [dev-dependencies]
 criterion = { version = "0.5", features = ["async_futures"] }
 datafusion-functions-aggregate = { workspace = true }
+datafusion-functions-window = { workspace = true }
```

Comment:
some day I hope we can remove these dependencies (so we can make testing `physical-plan` faster, but not a part of this PR